src/services/qix/schema.js

```diff
@@ -115,6 +115,14 @@ class Schema {
           enumerable: true,
           value: delta,
         },
+        type: {
+          enumerable: true,
+          value: typeKey,
+        },
+        customType: {
```

Comment:
The Engine API returns these as `qGenericType` (barchart) and `qType` (genericobject), I think it would be a good idea to use that naming here too.